src/main/java/org/mockito/verification/VerificationData.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2007 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
-package org.mockito.internal.verification.api;
+package org.mockito.verification;
```

Comment:
Very nice catch, thank you! I'll take a stab at exposing this type and report back.